cli-tests/test_manage_colors_in_output.sh

```
@@ -1,10 +1,10 @@
 test_no_color_if_no_terminal() {
-	assert_fail "start_misconfigured_amaru_daemon | has_escape_sequence" \
+	assert_fail "start_misconfigured_amaru_run_cmd | has_escape_sequence" \
 		"Found escape characters in output but we're not running in a terminal"
 }
 
 test_color_if_in_terminal() {
-	assert "emulate_terminal "$start_misconfigured_amaru_daemon_cmd" | has_escape_sequence" \
+	assert "emulate_terminal "$start_misconfigured_amaru_run_cmd" | has_escape_sequence" \
```

Shellcheck (0.11.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)